_includes/pbc/all/install-features/202304.0/install-the-customer-account-management-feature.md

```diff
@@ -0,0 +1,1198 @@
+
+
+This document describes how to integrate the [Customer Account Management](/docs/scos/user/features/{{site.version}}/customer-account-management-feature-overview/customer-account-management-feature-overview.html) feature into a Spryker project.
+
+{% info_block errorBox "Included features" %}
+
+The following feature integration guide expects the basic feature to be in place. The current feature integration guide only adds:
+* Redirect support for Customer login functionality.
+* Password set and reset console commands for customers.
+* Double opt-in for customer registration.
+* OAuth
+* Improved password security
+* Session validation
+
+{% endinfo_block %}
+
+## Install feature core
+
+Follow the steps below to install the Customer Account Management feature core.
+
+### Prerequisites
+
+To start feature integration, integrate the required features:
+
+| NAME         | VERSION          | INTEGRATION GUIDE                                                                                                                    |
+|--------------|------------------|--------------------------------------------------------------------------------------------------------------------------------------|
+| Spryker Core | {{site.version}} | [Spryker Core feature integration](/docs/scos/dev/feature-integration-guides/{{site.version}}/spryker-core-feature-integration.html) |
+
+### 1) Install the required modules using Composer
+
+Install the required modules:
+
+```bash
+composer require spryker-feature/customer-account-management: "{{site.version}}" spryker/oauth-customer-connector:"^1.6.0" --update-with-dependencies
+```
+
+{% info_block warningBox "Verification" %}
+
+Ensure that the following modules have been installed:
+
+| MODULE                  | EXPECTED DIRECTORY                           |
+|-------------------------|----------------------------------------------|
+| Customer                | vendor/spryker/customer                      |
+| CustomerGroup           | vendor/spryker/customer-group                |
+| CustomerNote            | vendor/spryker/customer-note                 |
+| CustomerNoteGui         | vendor/spryker/customer-note-gui             |
+| CustomerStorage         | vendor/spryker/customer-storage              |
+| CustomerValidationPage  | vendor/spryker-shop/customer-validation-page |
```

Comment:
this module and related to it things should be moved to respective section about frontend